### ISSUE TITLE
Method to access fontsize

### ISSUE THREAD

**Issue by agargara** [pdfminer/pdfminer.six#202]
I want to extract the font size of a given LTTextLine object.
My workaround has been to edit the `__init__` method of `class LTChar` to add the line:

`self.fontsize = fontsize`

And then later extract the fontsize of text like this:

```
# o is an instance of LTTextLine
for c in  o._objs:
    if isinstance(c, pdfminer.layout.LTChar):
        fontsize = int(round(c.fontsize))
```

Is there another method to access fontsize? 
If not, would it be possible to add one?

**Comment by jbarlow83** [comment]
`self.fontsize` is the font size set by the PDF content stream, but that's not the *effective font size* after you account for the current graphics state. The actual font size is closer to the height of the textline (`o.size` in your example).

It is quite possible for characters with different font sizes to be on the same textline - superscripts, for example.

**Comment by agargara** [comment]
Thanks jbarlow83, I hadn't taken superscripts into account.
I think this should really be considered two separate feature requests:

1. Make LTChar.fontsize a public instance variable. (easy)
2. Add a method LTChar.get_size() to get the actual pixel size of a given character. (more difficult)

**Comment by jackyjingyi** [comment]
> `self.fontsize` is the font size set by the PDF content stream, but that's not the _effective font size_ after you account for the current graphics state. The actual font size is closer to the height of the textline (`o.size` in your example).
> 
> It is quite possible for characters with different font sizes to be on the same textline - superscripts, for example.

would it be problematic if aggregate characters into textline or textbox by not only their coordinates but also their fontsize ( or maybe basefontname)?

**Comment by kuldeepnpatel** [comment]
Hello - Thanks. I want fontsize with the word also. which word has what size. c.size is returning the size but for which word how much? Please, help with this. Thanks

**Comment by pietermarsman** [comment]
You can use `LTChar.size` for this. 

(Previously this sometimes returned the wrong value but it was fixed in #348)
